.github/workflows/chart-test.yaml

```diff
@@ -79,6 +79,29 @@ jobs:
         if: steps.ct-lint.outcome == 'failure' || steps.ct-lint-all.outcome == 'failure'
         run: exit 1
 
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v4.2.0
+        with:
+          version: v3.6.3
+
+      - name: Install helm unittest plugin
+        run: helm plugin install --version v0.6.3  https://github.com/helm-unittest/helm-unittest.git
+
+      - name: Run chart unit tests
+        run: |
+          for dir in $(ls -d charts/*/); do
+            pushd $dir; helm dependencies update; popd
+            helm unittest $dir
+          done
+
```

Comment:
We need to add some alerting to the tests , similar to what we do for the install and lint tests: https://github.com/rstudio/helm/blob/884427b511575bd2bf7ae1149e7db1106f12b8c9/.github/workflows/chart-test.yaml#L154-L173